mlir_graphblas/algorithms.py

```diff
@@ -1,14 +1,7 @@
 import numpy as np
 from typing import List
-from .functions import (
-    ConvertLayout,
-    MatrixSelect,
-    MatrixReduceToScalar,
-    Apply,
-    MatrixMultiply,
-)
 from mlir_graphblas.mlir_builder import MLIRVar, MLIRFunctionBuilder
```

Comment:
This wasn't part of this PR, but `MLIRVar` isn't actually used in this file, so we should consider removing it.